geckomat/model_adapter/ModelAdapter.m

```diff
@@ -17,7 +17,12 @@
         %end
         
 		function model = manualModifications(obj,model) %default is to do nothing
-		end
+        end
+        
+        %The genes returned here should match the gene id set in the parameter params.uniprotGeneIdField
+        function genes = getUniprotCompatibleGenes(obj,model)
+            genes = model.genes;
+        end
```

Comment:
This should also be in `YeastGEMAdapter.m`, right?

Reply:
So, if I remember correctly, the yeast model follows the standard behavior, so there is no need to specialize any code for that and we can therefore rely on the base class implementation. I have been running them both and they worked, so I think it should be ok.

Reply:
There was some kind of special case for Human-GEM that I had to deal with.

Reply:
But it should be included, as `makeEcModel` is calling `uniprotCompatibleGenes = modelAdapter.getUniprotCompatibleGenes(model);`, and for simples cases this should just output `model.genes` as is done in this template adapter.